### ISSUE TITLE
How you prepare the bounding box when transfeering from CMU dataset to Human3.6m?

### ISSUE THREAD

**Issue by mengmenm** [karfly/learnable-triangulation-pytorch#43]
Amazing jobs, guys.

I am interested in the transformation part mentioned in the paper. And very curious about the bounding box you used when transferring the pre-trained CMU model to the Human3.6m dataset. 
So which bbox you use when transfer:  the original bbox provided by human3.6m or the modified human3.6m bbox which matches the pre-trained model(CMU dataset)?

**Comment by karfly** [comment]
Hi, @mengmenm.
Thank you for a good question. During training on CMU Panoptic dataset we used Mask R-CNN bboxes of humans. When we transfer out models from CMU Panoptic to Human3.6M we use bboxes provided by Human3.6M dataset (GT bboxes). We found out that our models don't overfit to the type of the bbox.

**Comment by mengmenm** [comment]
hi, @karfly 
thank you so much.
